platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/EmbeddedSolrFactory.java

```diff
@@ -46,167 +42,176 @@
  * </ul>
  */
 public class EmbeddedSolrFactory implements SolrClientFactory {
-
   private static final Logger LOGGER = LoggerFactory.getLogger(EmbeddedSolrFactory.class);
 
-  private static final String DEFAULT_EMBEDDED_CORE_NAME = "embedded";
+  @VisibleForTesting
+  static final Map<EmbeddedSolrFiles, IndexSchema> INDEX_CACHE = new ConcurrentHashMap<>();
```

Comment:
❓Exposing the `INDEX_CACHE` map for testing purposes makes the tests pretty brittle. Instead of doing that, could we have a package-private method to "reset" the map (since there's no easy way of doing that other than forcing a class unload/reload or make this member variable non-static), and verify the map state through side-effects, e.g., something happens or doesn't happen because it's (not) already in the map?